pkg/util/metadata/metadata.go

```diff
@@ -101,6 +101,10 @@ func getConfigDrivePath(metadataVersion string) string {
 	return fmt.Sprintf(configDrivePathTemplate, metadataVersion)
 }
```

Comment:
@jichenjc  I think we should have an interface for metadata package, it is required for testing, and while doing csi sanity testing for cinder csi driver i realized  it is must

Reply:
@adisky I have similar thought, let me try to work on it soon